### ISSUE TITLE
import speed is very slow

### ISSUE THREAD

**Issue by yuqin11** [JanusGraph/janusgraph#2041]
Please post all questions and issues on [janusgraph-users](https://groups.google.com/forum/#!forum/janusgraph-users)
before opening a GitHub issue. Your questions will reach a wider audience there,
and if we confirm that there is a bug, then you can open a new issue.

Please include configurations and logs if available.
`
gremlin.graph= org.janusgraph.core.JanusGraphFactory
storage.backend= cql
storage.cql.keyspace=graphtest
query.batch= true
query.force-index= true
storage.cql.compression= true
storage.cql.compression-block-size=4
storage.cql.read-consistency-level=LOCAL_ONE
storage.cql.write-consistency-level=LOCAL_QUORUM
storage.cql.local-core-connections-per-host=2
storage.cql.local-max-connections-per-host= 2
storage.cql.replication-factor=1
storage.hostname=9.109.52.166
ids.block-size= 10000000
schema.default=none
storage.batch-loading=true
storage.buffer-size=2048
index.ES.backend= elasticsearch
index.ES.hostname=9.109.52.166
index.ES.elasticsearch.client-only= true
index.ES.index-name= graphtest
index.ES.elasticsearch.create.ext.index.number_of_shards= 4
index.ES.elasticsearch.create.ext.index.number_of_replicas= 0
index.ES.elasticsearch.create.ext.index.refresh_interval= 600s
log.tx.key-consistent=true
cache.db-cache-size= 0.0
cache.db-cache-time= 0
graph.replace-instance-if-exists=true
`
when id use janus package to import data properties is above code is below .

For confirmed bugs, please report:
- Version:  0.3.1
- Storage Backend:  cql
- Mixed Index Backend: es
- Mailing list Thread URL: 
- Steps to Reproduce: 

### my data import code

```
// 基础读取和导入资源
            startNs = System.nanoTime();
            JanusGraph graph = open(configurationFile, graphName);
            GraphTraversalSource g = graph.traversal();
            janusTime += System.nanoTime() -startNs;
            reader = new BufferedReader(new InputStreamReader(fs.open(path)));
            // 表头
            String firstLine = reader.readLine();
            HashMap<Integer, String> rowMap = parseRowMap(firstLine, elementType, labelName);

            if(rowMap == null){
                logger.error(" parse file header of file {} failed, pass this file", fileName);
                return;
            }

            // 按行处理
            String line;
            Long count = new Long(0);
            if (elementType.equals(vertexPrefix)) {
                HashMap<String, String> elementMap = new HashMap<>();
                while ((line = reader.readLine()) != null) {
                    // 初始化 节点map
                    elementMap.clear();
                    // 每行数据以tab分割，且列数必须等于表头
                    String[] lineValues = line.split(dataSeperator);
                    if(lineValues.length != rowMap.size()){
                        logger.warn("pass invalid data {} th line of file {}:, do not import this line", count, fileName);
                        count += 1;
                        continue;
                    }
                    // 解析一行数据
                    for(int i = 0; i < lineValues.length; i++){
                        Integer index = Integer.valueOf(i);
                        elementMap.put(rowMap.get(index), lineValues[i]);
                    }
                    elementMap.put(typeForKgSchema, labelName);
                    // 导入
                    importOneVertex(elementMap, g, labelName, count);
                    count += 1;
                    if((count + 1) % batchSize == 0){
                        Long startNs1 = System.nanoTime();
                        g.tx().commit();
                        commitTime += System.nanoTime() - startNs1;
                    }
                    if (count % 10000 == 0){
                        logger.info("processed {} lines " + elementType, count);
                    }
                }
                Long startNs1 = System.nanoTime();
                g.tx().commit();
                commitTime += System.nanoTime() - startNs1;
}
}
```

**Comment by farodin91** [comment]
Are you able to perform this in parallel? Here would good to connect remotely since gremlin server can open up multiple JanusGraph instances.
Are you able to show the same results on JanusGraph 0.5? We are no longer support 0.3. 

Your block size is 10 000 000 which should be the number of created ids in one hour. Futhermore, you say 3M/min which not correlate. So, i would first of all increase the block size for example 3 000 000 * 60, see here https://docs.janusgraph.org/advanced-topics/bulk-loading/.

Could check the jvm heap?
